src/qrl/core/Miner.py

```diff
@@ -18,6 +18,12 @@
 
 
 class Miner(Qryptominer):
+    """
+    QRL can be solo-mined and pool-mined. This class handles solo-mining, where
```

Comment:
"This class handles solo-mining" <-- Incorrect information. This class is for all kinds of mining. Even mining pool use this same class to get the block template for mining.